packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -105,119 +130,116 @@ export default {
 		padding-top: var(--unit-1);
 		padding-bottom: var(--unit-1);
 		line-height: var(--default-button-line-height);
-		&.icon {
+
+		&.kt-button--has-icon {
 			align-items: baseline;
 			text-align: left;
+
 			.yoco {
 				position: relative;
```

Comment:
why relativee

can u maybe flex the contents, and that's it?

Reply:
It’s for the crazy `isMultiline` BS. Really don’t want to touch it, sorry.